modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/disruptor/delivery/ConcurrentContentReadTaskBatchProcessor.java

```diff
@@ -63,7 +64,7 @@
      * @param eventHandler    is the delegate to which events are dispatched.
      * @param turn            is the value of, sequence % groupCount this batch processor process events. Turn must be
      *                        less than groupCount
-     * @param groupCount      total number of concurrent batch processors for the event type
+     * @param groupCount      total number of concurrent batch processors for the deliveryEventArrayList type
```

Comment:
Yah, renamed them probably they were the refactored words, when I try to rename one place